packages/core/svelte/src/workSpace.ts

```diff
@@ -0,0 +1,14 @@
+import { writable } from 'svelte/store';
+import type { Program, Provider, web3 } from '@project-serum/anchor';
```

Comment:
Unsure about the best way to handle this scenario. We only need the `@project-serum/anchor` package for typing, but seems unreasonable to list this as `devDependency`

Reply:
I think that in this case, make totally sense for me to list this just as `devDependency`.

Reply:
@cogoo `@project-serum/anchor` this will be removed since this file is removed too.